worldedit-core/src/main/java/com/sk89q/worldedit/util/LocatedBlock.java

```diff
@@ -22,46 +22,37 @@
 import com.sk89q.worldedit.math.BlockVector3;
 import com.sk89q.worldedit.world.block.BaseBlock;
 
-import java.util.Objects;
-
 import static com.google.common.base.Preconditions.checkNotNull;
 
 /**
  * Represents a block located at some position.
  */
-public final class LocatedBlock {
-
-    private final BlockVector3 location;
-    private final BaseBlock block;
+public record LocatedBlock(BlockVector3 location, BaseBlock block) {
 
     public LocatedBlock(BlockVector3 location, BaseBlock block) {
```

Comment:
Can drop argument list here. 
```suggestion
     public LocatedBlock {
```